code_annotations/cli.py

```diff
@@ -171,3 +166,49 @@ def static_find_annotations(config_file, source_path, report_path, verbosity, li
     except Exception as exc:  # pylint: disable=broad-except
         click.echo(traceback.print_exc())
         fail(str(exc))
+
+
+@entry_point.command("generate_docs")
+@click.option(
+    '--config_file',
+    default='.annotations',
+    help='Path to the configuration file',
+    type=click.Path(exists=True, dir_okay=False, resolve_path=True)
+)
+@click.argument("report", type=click.File('r'))
```

Comment:
Eventually we were hoping to be able to feed in multiple report files and have them rolled together to gather output from the various tools (or various repos) together in the doc. Not a blocker for this, but if you find a minute to add it that would be grand.